tools/ncf.py

```diff
@@ -16,6 +16,8 @@
 import sys
 import os
 import codecs
+import ncf_constraints
+from pprint import pprint
```

Comment:
OK, fine. It's nice to answer comments though, even if you don't agree with them :)